dll/win32/netapi32/group_new.c

```diff
@@ -646,6 +640,9 @@ NetGroupDel(
         goto done;
     }
 
+    /* A successful delete invalidates the handle */
+    GroupHandle = NULL;
+
```

Comment:
Now the code below is dead code because you manually invalidated `GroupHandle` (and wrongly so).

Reply:
How is my change wrong? How does it create dead code?